src/drivers/esp8266_drv.c

```diff
@@ -736,6 +741,18 @@ static void set_fast_baud()
 	cmd_set_check(CHECK_WIFI_DEVICE);
 }
 
+static void hard_reset_wifi(void)
```

Comment:
This fixed a bug where, if we do a hard reset, we don't set our baud rate back to factory default of 115200.  This fixes this.